ebos/eclwriter.hh

```diff
@@ -333,6 +338,19 @@ public:
     void endRestart()
     {}
 
+    std::size_t previousRestartOutput() const
+    {
+        auto previous = std::size_t{0};
+        if (this->eclIO_ != nullptr) {
+            previous = this->eclIO_->previousRestartOutputStep();
+        }
+
+        this->simulator_.vanguard().gridView().comm()
```

Comment:
It is only misleading to broadcast this - it is 100% based on the input configuration and that is already globally distributed.